packages/k10/generated-changes/overlay/questions.yaml

```
@@ -0,0 +1,295 @@
+questions:
+# ========================
+# SECRETS And Configuration
+# ========================
+
+### AWS Configuration
+
+- variable: secrets.awsAccessKeyId
+  description: "AWS access key ID (required for AWS deployment)"
+  type: password
+  label: AWS Access Key ID
+  required: false
+  group: "AWS Configuration"
+
+- variable: secrets.awsSecretAccessKey
+  description: "AWS access key secret (required for AWS deployment)"
+  type: password
+  label: AWS Secret Access Key
+  required: false
+  group: "AWS Configuration"
+
+- variable: secrets.awsIamRole
+  description: "ARN of the AWS IAM role assumed by K10 to perform any AWS operation."
+  type: string
+  label: ARN of the AWS IAM role
+  required: false
+  group: "AWS Configuration"
+
+- variable: awsConfig.assumeRoleDuration
+  description: "Duration of a session token generated by AWS for an IAM role"
+  type: string
+  label: Role Duration
+  required: false
+  default: ""
+  group: "AWS Configuration"
+
+- variable: awsConfig.efsBackupVaultName
+  description: "Specifies the AWS EFS backup vault name"
+  type: string
+  label: EFS Backup Valut Name
```

```suggestion
  label: EFS Backup Vault Name
```

A typo I will correct that